src/components/TasksTable/index.jsx

```diff
@@ -251,13 +251,15 @@ export default class TasksTable extends Component {
     this.setQuery({});
   };
 
-  handleDrawerOpen = ({ target: { name } }) => {
+  handleDrawerOpen = ({ currentTarget: { name } }) => {
     memoizeWith(
       name => name,
       name =>
         this.setState({
           drawerOpen: true,
-          drawerItem: this.props.items.find(item => item.summary === name),
+          drawerItem: this.props.items.find(
+            item => sort(item.summary, name) === 0
```

Comment:
What's wrong with `item.summary === name`?

Reply:
it fails the travis build. i tried it 2 times.
At first, i wonder why the travis fail, then i randomly change the find using sort method, so that i can force to rerun the travis (i remember when we add testing it also fail at first then you rerun it somehow and it works) , then, it runs successfully, but then i do

`g reset --hard HEAD^` 
`g push origin fix/handle-drawer-open-on-chrome-and-mobile -f`

to remove the last commit about the find using sort then it fail again

Reply:
Can you try again (remove `sort`) and send me the link to the failed travis build, please? I'll take a look.

Reply:
ok removed! this is the link https://travis-ci.com/mozilla-frontend-infra/codetribute/builds/79747509

Reply:
Seems to be okay now.